docker-compose.yml

```diff
@@ -0,0 +1,73 @@
+version: "3.8"
+
+services:
+  rocc-database:
+    image: sagebionetworks/mongo:4.4.4
+    container_name: rocc-db
+    restart: always
+    environment:
+      - MONGO_INITDB_ROOT_USERNAME
+      - MONGO_INITDB_ROOT_PASSWORD
+      - MONGO_INITDB_DATABASE
+      - MONGO_USERNAME
+      - MONGO_PASSWORD
+    networks: 
+      - rocc-networks
+
+  rocc-service:
+    image: sagebionetworks/rocc-service:0.2.1
+    container_name: rocc-service
+    restart: always
+    ports:
+      - "8080:8080"
+    environment:
+      - SERVER_PROTOCOL
+      - SERVER_DOMAIN
+      - SERVER_PORT=8080
+      - DB_PROTOCOL
+      - DB_DOMAIN=rocc-database
+      - DB_PORT
+      - DB_DATABASE
+      - DB_USERNAME
+      - DB_PASSWORD
+    networks: 
+      - rocc-networks
+    depends_on: 
+      - rocc-database
+
+  rocc-app:
+    image: rocc/rocc-app:latest
```

Comment:
The image should be `sagebionetworks/rocc-app`. We should also include the version to run instead of `latest`.

Reply:
- 323eeab: change `rocc/rocc-app:latest` to `sagebionetworks/rocc-app`
- a203f1a: add images' versions  from `.env` variables